field.go

```diff
@@ -368,7 +368,22 @@ func Stack(key string) Field {
 	// from expanding the zapcore.Field union struct to include a byte slice. Since
 	// taking a stacktrace is already so expensive (~10us), the extra allocation
 	// is okay.
-	return String(key, takeStacktrace())
+	return String(key, takeStacktrace(0))
```

Comment:
Rather than having Stack and StackSkip be nearly the same, I think it may be better if Stack called StackSkip.

```
	return StackSkip(1) // skip Stack
```